backend/src/openarchiefbeheer/zaken/tasks.py

```diff
@@ -20,19 +25,32 @@ def retrieve_and_cache_zaken_from_openzaak() -> None:
         response = zrc_client.get(
             "zaken",
             headers={"Accept-Crs": "EPSG:4326"},
-            params={"expand": "resultaat,resultaat.resultaattype,zaaktype"},
+            params={
+                "expand": "resultaat,resultaat.resultaattype,zaaktype,rollen",
+                "archiefnominatie": "vernietigen",
+            },
+            timeout=settings.REQUESTS_DEFAULT_TIMEOUT,
         )
         response.raise_for_status()
 
         data_iterator = pagination_helper(
-            zrc_client, response.json(), headers={"Accept-Crs": "EPSG:4326"}
+            zrc_client,
+            response.json(),
+            headers={"Accept-Crs": "EPSG:4326"},
+            params={
+                "expand": "resultaat,resultaat.resultaattype,zaaktype,rollen",
+                "archiefnominatie": "vernietigen",
+            },
+            timeout=settings.REQUESTS_DEFAULT_TIMEOUT,
         )
 
     with transaction.atomic():
         # Removing existing cached zaken
         Zaak.objects.all().delete()
 
-        for data in data_iterator:
+        for index, data in enumerate(data_iterator):
+            logger.debug("Retrieved page %s.", index + 1)
```

Comment:
Is adding the log intentional? Just checking.

Reply:
Yes, I found it nice to see how long each page was taking and seeing that there was some progress (requesting each page is very slow). So I added logging with debug level, so it should only be displayed if configured to display debug logs.